### PR TITLE
chore(docs): corrected recommended settings.php location references

### LINKED ISSUE #8664: Update docs to point to new settings location
@ewinslow i believe docs still says the settings should be in engine/settings.php... do we want to change that?

Related to #8652

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #8664

### CODE REVIEW

File: docs/admin/duplicate-installation.rst [hunk 1]
```diff
@@ -92,10 +92,10 @@ If you don't have shell access to your server and have to ftp the data, you may
    
    You also need to delete the views cache on the test server after the copy process. This is a directory called ``views_simplecache`` in your data directory and the directory called ``system_cache`` .
 
-Edit engine/settings.php
+Edit settings.php
 ========================
```
Comment (mrclay) [inline docs/admin/duplicate-installation.rst]: `=` should match new heading length

File: docs/intro/install/cloud9.rst [hunk 2]
```diff
@@ -26,11 +26,11 @@ Run the following in cloud9's terminal:
     rm -rf * # Clear out the c9 hello-world stuff
     composer create-project elgg/elgg:~1.11 . # the hotness
     cp install/config/htaccess.dist .htaccess
-    cp engine/settings.example.php engine/settings.php
+    cp engine/settings.example.php settings.php
```
Comment (mrclay) [inline docs/intro/install/cloud9.rst]: Shouldn't we move settings.example.php to root, too? cc @ewinslow

**Comment by mrclay** [comment]
Besides comment, LGTM

File: docs/intro/install/cloud9.rst (hunk 2 above)
Comment (jdalsem) [inline docs/intro/install/cloud9.rst]: Or should it be moved to install/config/settings.example.php?? that is where htaccess.dist is also located

**Comment by jdalsem** [comment]
Anything else?

**Pull request merged**